tests/lax_scipy_test.py

```diff
@@ -59,6 +59,7 @@ def op_record(name, nargs, dtypes, rng_factory, test_grad, nondiff_argnums=(), t
     op_record("gammaincc", 2, float_dtypes, jtu.rand_positive, True),
     op_record("erf", 1, float_dtypes, jtu.rand_small_positive, True),
     op_record("erfc", 1, float_dtypes, jtu.rand_small_positive, True),
+    op_record("erfcx", 1, float_dtypes, jtu.rand_small_positive, True),
```

Comment:
For small numbers, this will be *very* close to erfc. I'd try with `jtu.rand_positive` or even `jtu.rand_default` (though for larger negative numbers, the output diverges pretty quickly)

Reply:
Sure, let me try it first. Thanks!